onnxscript/function_libs/torch_lib/ops/prims.py

```diff
@@ -837,7 +837,7 @@ def prims_view_of(a: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
-@torch_op("prims::where", traceable=True)
```

Comment:
`traceable` is now fully retired? Or we still rely it on somewhere? If not, we can delete the parameter.

Reply:
Yes. We can delete the param

Reply:
Done